earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -26,11 +26,28 @@
 # lines below.
 
 Import('third_party_env')
+import os
 
 gdal_version = 'gdal-2.1.2'
 openjpeg_version = 'openjpeg-v2.1.2-linux-x86_64'
 # kakadu_version = 'v5_2_6-00418C'
 # geoexpress_version = 'MrSID_DSDK-8.5.0.3422'
+
+# Check if MrSID decoding library is installed and get the decoding SDK path
+mrSIDLibName = 'libltidsdk.so'
+mrSIDDSDKPath = ""
+findLibraryCommand = "/sbin/ldconfig -p | grep %s | tr ' ' '\n' | grep /" %mrSIDLibName
+mrSIDlibraryFullPath=os.popen(findLibraryCommand).read()
+if not mrSIDlibraryFullPath:
+   print("mrSID library was not found ! GDAL will be built without mrSID raster format support.")
```

Comment:
I don't know if this print statement is necessary.  Building without MrSID is the default, so I'm not sure if we want to notify people about it.

Reply:
I think its helpful if people actually have the MrSIDDSK in some other path and expect that to work, unless that is absolutely not a case ever( meaning do we gurantee that there would always be same directory/ filepath or a symlink regardless of OS platform ?)

Reply:
MrSID DSDK can be 'installed' in any folder as the user choose to. The only requirement is that location is discover-able by 'ldconfig'. This means that 'ldconfig' cache need to be refreshed after installing MrSID DSK.

Reply:
@tst-lsavoie : i can remove the print statement when mrSID DSDK library was not found and keep the one when the library is found.

Reply:
done